sklearn/metrics/tests/test_common.py

```diff
@@ -1113,9 +1147,14 @@ def test_no_averaging_labels():
                                [y_true_multilabel, y_pred_multilabel]]:
             if name not in MULTILABELS_METRICS and y_pred.ndim > 1:
                 continue
+            if name in NONE_AVERAGE_UNDEFINED_MULTICLASS and y_pred.ndim < 2:
```

Comment:
Here is an example of where I've used `NONE_AVERAGE_UNDEFINED_MULTICLASS`.

Reply:
Well you can either drop "multiclass" here or you can implement average=None